docs/conda.md

```diff
@@ -163,10 +163,46 @@ https://conda.anaconda.org/conda-forge/linux-64/libgcc-ng-13.2.0-h77fa898_7.cond
 # .. and so on
 ```
 
-To use a Conda lock file with Nextflow, set the `conda` directive to the path of the lock file.
+To use a Conda lock file with Nextflow, set the `conda` directive to the path of the lock file:
 
-:::{note}
-Conda lock files must be a text file with the `.txt` extension.
+```nextflow
+process hello {
+  conda '/path/to/spec-file.lock'
+
+  script:
+  """
+  your_command --here
+  """
+}
+```
+
+You can also specify Conda lock files using a remote URL (HTTP, HTTPS, S3, Google Cloud Storage, Azure Blob Storage, etc.):
+
+```nextflow
+process hello {
+  conda 'https://wave.seqera.io/v1alpha1/builds/bd-12345/condalock'
+
+  script:
+  """
+  your_command --here
+  """
+}
+```
+
+```nextflow
+process hello {
+  conda 's3://my-bucket/conda/env.lock'
+
+  script:
+  """
+  your_command --here
+  """
+}
+```
```

Comment:
I think it's probably a bit excessive to spell out the s3 stuff as well, this is probably fine:
````suggestion
You can also specify Conda lock files using a remote URL:

```nextflow
process hello {
  conda 'https://wave.seqera.io/v1alpha1/builds/bd-12345/condalock'

  script:
  """
  your_command --here
  """
}
```

````